sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/RecognizeBusinessCardsOptions.java

```diff
@@ -0,0 +1,106 @@
+@Fluent
+public final class RecognizeBusinessCardsOptions {
+    private static final Duration DEFAULT_POLL_INTERVAL = Duration.ofSeconds(5);
+    private FormContentType contentType;
+    private boolean includeFieldElements;
+    private Duration pollInterval = DEFAULT_POLL_INTERVAL;
+    private String locale;
+
+    /**
+     * Get the type of the form. Supported Media types including .pdf, .jpg, .png or .tiff type file stream.
+     *
+     * @return the {@code contentType} value.
+     */
+    public FormContentType getContentType() {
+        return contentType;
+    }
+
+    /**
+     * Get the boolean which specifies if to include form element references in the result.
+     *
+     * @return the {@code includeFieldElements} value.
+     */
+    public boolean isFieldElementsIncluded() {
+        return includeFieldElements;
+    }
+
+    /**
+     * Get the duration between each poll for the operation status. If none is specified, a default of
+     * 5 seconds is used.
+     *
+     * @return the {@code pollInterval} value.
+     */
+    public Duration getPollInterval() {
+        return pollInterval;
+    }
+
+    /**
+     * Set the type of the form. Supported Media types including .pdf, .jpg, .png or .tiff type file stream.
+     *
+     * @param contentType the provided form content type.
+     *
+     * @return the updated {@code RecognizeBusinessCardOptions} value.
+     */
+    public RecognizeBusinessCardsOptions setContentType(final FormContentType contentType) {
+        this.contentType = contentType;
+        return this;
+    }
+
+    /**
+     * Set the boolean which specifies if to include form element references in the result.
+     *
+     * @param includeFieldElements the boolean to specify if to include form element references in the result.
+     *
+     * @return the updated {@code RecognizeBusinessCardOptions} value.
+     */
+    public RecognizeBusinessCardsOptions setFieldElementsIncluded(final boolean includeFieldElements) {
+        this.includeFieldElements = includeFieldElements;
+        return this;
+    }
+
+    /**
+     * Set the duration between each poll for the operation status. If none is specified, a default of
+     * 5 seconds is used.
+     *
+     * @param pollInterval the duration to specify between each poll for the operation status.
+     *
+     * @return the updated {@code RecognizeBusinessCardOptions} value.
+     */
+    public RecognizeBusinessCardsOptions setPollInterval(final Duration pollInterval) {
+        this.pollInterval = pollInterval == null ? DEFAULT_POLL_INTERVAL : pollInterval;
+        return this;
+    }
+
+    /**
+     * Get the locale value.
+     *
+     * @return the locale value.
+     */
+    public String getLocale() {
+        return locale;
+    }
+
+    /**
+     * Set the locale value.
+     *
+     * @param locale the locale value.
+     *
+     * @return the locale value.
+     */
+    public RecognizeBusinessCardsOptions setLocale(final String locale) {
```

Comment:
can you include the dfault value is US

Reply:
I wonder if we should create another link for the supported locales and add it to the docstings here to help users.
Currently the supported locales for business cards are here (no ms.docs link :( ): https://westus2.dev.cognitive.microsoft.com/docs/services/form-recognizer-api-v2-1-preview-1/operations/AnalyzeBusinessCardAsync

Reply:
If we don't pass a empty/null locale  value to service, does service uses 'en-US' as the default value?

Reply:
Yes, but my understanding is that they don't want us to set a default. They will like us to pass nothing and let the service figure it out

Reply:
@mssfang and @iscai-msft The locale functionality will be updated in this PR - https://github.com/Azure/azure-sdk-for-java/pull/16027

@maririos Can we use this link for supported locales - https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/concept-receipts#supported-locales

Reply:
we could although this are locales for receipts. Still better than nothing.
I did ask to the service team in our Teams channel => https://teams.microsoft.com/l/message/19:88f2d9dea2344075b5d4bc34d82a3d1c@thread.skype/1602013362187?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1602013362187&teamName=Azure%20SDK&channelName=Service%20-%20Form%20Recognizer&createdTime=1602013362187

Reply:
>Yes, but my understanding is that they don't want us to set a default. They will like us to pass nothing and let the service figure it out

I stand corrected. Looks like for locale we could set the default to `en-US` like @samvaity  did in her PR :)

Reply:
@maririos @samvaity wouldn't it be better to not set any default on our side, and allow the service to handle the default case if no locale is passed in?

Reply:
adding to this discussion that paul strongly recommends not hardcoding a default on the SDK side